core/trino-main/src/main/java/io/trino/event/QueryMonitor.java

```diff
@@ -140,7 +141,10 @@ public void queryCreatedEvent(BasicQueryInfo queryInfo)
         eventListenerManager.queryCreated(
                 new QueryCreatedEvent(
                         queryInfo.getQueryStats().getCreateTime().toDate().toInstant(),
-                        createQueryContext(queryInfo.getSession(), queryInfo.getResourceGroupId(), queryInfo.getQueryType()),
+                        createQueryContext(queryInfo.getSession(),
```

Comment:
wrap next line

Reply:
Yeah - I wonder if you can teach IntelliJ to do that